Lib/glyphsLib/classes.py

```diff
@@ -3760,6 +3765,46 @@ def _bracket_info(self):
         else:
             return axis_index, bracket_crossover, None
 
+    def _is_brace_layer(self):
+        if self.parent.parent.format_version > 2:
+            return "coordinates" in self.attributes  # Glyphs 3
+        # Glyphs 2
+        return "{" in self.name and "}" in self.name and ".background" not in self.name
+
+    def _brace_coordinates(self):
+        if not self._is_brace_layer():
+            return None
+
+        if self.parent.parent.format_version > 2:
+            return self.attributes["coordinates"]  # Glyphs 3
+
+        # Glyphs 2
+        name = self.name
+        coordinates = name[name.index("{") + 1 : name.index("}")]
+        return [float(c) for c in coordinates.split(",")]
+
+    def _is_color_palette_layer(self):
+        if self.parent.parent.format_version > 2:
+            return "colorPalette" in self.attributes  # Glyphs 3
+        return self.name.startswith("Color ")  # Glyphs 2
```

Comment:
maybe a regex would be better, so you only match `^Color (?:\*|\d+)$`, instead of any layer that starts with "Color "

Reply:
Done.